README.md

```diff
@@ -77,6 +77,11 @@ resource "cyral_integration_datadog" "datadog" {
     api_key = "datadog-api-key"
 }
 
+resource "cyral_integration_pager_duty" "pd" {
+  name = "my-pager-duty-integration"
+  api_token = "my-pd-api-token"
+}
+
```

Comment:
```suggestion
```

The example documentation here is the minimum for some different resources. This integration resource is similar to the others, so not necessary here.